context_prop/build.gradle

```diff
@@ -5,7 +5,6 @@ dependencies {
     api libraries.grpc_context
 
     signature "org.codehaus.mojo.signature:java17:1.0@signature"
-    signature "net.sf.androidscents.signature:android-api-level-14:4.0_r4@signature"
```

Comment:
this needs to be API-14, api depends on this.

Reply:
done